cmake/FindComputeCpp.cmake

```diff
@@ -89,11 +89,12 @@ function(build_spir exe_name spir_target_name source_file output_path)
             ${computecpp_device_compiler_defs}
             ${COMPUTECPP_USER_FLAGS}
             ${platform_specific_args}
-            $<$<BOOL:${include_directories}>:-I\"$<JOIN:${include_directories},\"\t-I\">\">
-            $<$<BOOL:${device_compile_definitions}>:-D$<JOIN:${device_compile_definitions},\t-D>>
-            $<JOIN:${device_compile_options},\t>
+            $<$<BOOL:${include_directories}>:-I\"$<JOIN:${include_directories},\"\;-I\">\">
```

Comment:
After being puzzled by the meaning of `\;` I looked at the CMake documentation. Ouch, it has really a meaning. :-)

Reply:
Add a comment explaining how it works along https://cmake.org/cmake/help/latest/manual/cmake-language.7.html#escape-sequences

Reply:
Isn't this because `;` is used for lists in CMake, so you need to escape that? Do we need to document that? I guess it's a bit strange, but so is large part of CMake :)

Reply:
Yes, that is fair.